docs/source/installation.rst

```diff
@@ -2,152 +2,125 @@ Installation
-+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+            pip install --upgrade pip
 
+            pip install ivadomed
 
-::
+    .. tab:: Repo Installation (Advanced or Developer)
 
-    git clone https://github.com/ivadomed/ivadomed.git
-    cd ivadomed
- 
+        Bleeding-edge developments are available on the project's master branch
+        on Github. Installation ``ivadomed`` from source:
 
-Step 3: Install PyTorch 1.5 and TorchVision
-+++++++++++++++++++++++++++++++++++++++++++
+        ::
 
-If you have a compatible NVIDIA GPU that supports CUDA, run the following command:
+            git clone https://github.com/ivadomed/ivadomed.git
 
-::
-   
-   pip install -r requirements_gpu.txt
+            cd ivadomed
 
+            pip install -e .
 
-If you do not have a compatible GPU, run the following installer to use ``ivadomed`` with CPU.
 
+Step 3: Install ``ivadomed`` with CPU or GPU Support, Install ``torch`` and ``torchvision``
+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 
-::
+.. tabs::
 
-    pip install -r requirements.txt
+    .. tab:: CPU Support
 
+        If you plan to run ``ivadomed`` on CPU only, run the following command:
+        ::
 
-Step 4: Install from release (recommended)
-++++++++++++++++++++++++++++++++++++++++++
+            pip install -r requirements.txt
 
-Install ``ivadomed`` and its requirements from
-`Pypi <https://pypi.org/project/ivadomed/>`__:
 
-::
+    .. tab:: Nvidia GPU Support
 
-    pip install --upgrade pip
-    pip install ivadomed
+        ``ivadomed`` requires CUDA11 to execute properly. If you have a nvidia GPU, try to look up its Cuda Compute Score `here <https://developer.nvidia.com/cuda-gpus>`_, which needs to be > 3.5 to support CUDA11. Then, make sure to upgrade to nvidia driver to be at least v450+ or newer.
 
-(Optional) Alternative Step 4 for Developers: Install from source
-+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+        If you have a compatible NVIDIA GPU that supports CUDA11 and with the right driver installed, try run the following command:
 
-Bleeding-edge developments are available on the project's master branch
-on Github. Installation procedure is the following at repository root:
+        ::
 
-::
+           pip install -r requirements_gpu.txt
 
-    cd ivadomed
-    pip install -e .
+Developer-only Installation Steps
++++++++++++++++++++++++++++++++++
 
+The additional steps below are only necessary for contributors to the ``ivadomed`` project.
 
-(Optional) Step 5 for Developers Install pre-commit hooks
-+++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+Additional Dependencies and ``pre-commit``
+-------------------------------------------
 
 We use ``pre-commit`` to enforce a limit on file size.
 After you've installed ``ivadomed``, install the hooks:
 
 ::
 
     pip install -r requirements_dev.txt
```

Comment:
I get this error when installing:
`ERROR: sphinx-tabs 3.2.0 has requirement docutils~=0.16.0, but you'll have docutils 0.17.1 which is incompatible.`

I'm not sure what the impact is, I was able to build the doc locally without errors.

Reply:
Interesting! It looks like `sphinx-tabs` only updated its requirements (`0.16 -> 0.17`) last month (https://github.com/executablebooks/sphinx-tabs/pull/144), so I'm guessing this hasn't made its way to a release yet.

That said, `pip` should be smart enough to resolve dependencies between all of the requirements in `requirements-dev.txt`. I'm surprised it's throwing an error here. :thinking: